pkg/env/prod.go

```diff
@@ -91,8 +95,10 @@ func newProd(ctx context.Context, log *logrus.Entry) (*prod, error) {
 
 		fpClientID: os.Getenv("AZURE_FP_CLIENT_ID"),
 
-		clusterGenevaLoggingEnvironment:   os.Getenv("MDSD_ENVIRONMENT"),
+		clusterGenevaLoggingAccount:       os.Getenv("CLUSTER_MDSD_ACCOUNT"),
+		clusterGenevaLoggingNamespace:     os.Getenv("CLUSTER_MDSD_NAMESPACE"),
 		clusterGenevaLoggingConfigVersion: os.Getenv("CLUSTER_MDSD_CONFIG_VERSION"),
+		genevaLoggingEnvironment:          os.Getenv("MDSD_ENVIRONMENT"),
```

Comment:
This name `genevaLoggingEnvironment` was `clusterGenevaLoggingEnvironment`.  Because it's the same env var, is there any reason why we are changing this name?